Paws4Me/Paws4Me/Controllers/Logic/PetSingleDetailsViewController.swift

```diff
@@ -20,7 +20,6 @@ class PetSingleDetailsViewController: UIViewController {
     var genderOfPet = ""
     var ageOfPet = ""
     var imagOfPet = ""
-    
     override func viewDidLoad() {
         super.viewDidLoad()
         petName.text=nameOfPet
```

Comment:
`petName.text = nameOfPet`

Also another small note here but normally when naming variables we try and avoid some of the smaller words like "and", "of", etc. 

Maybe a better name could be: `petName`

Please also consider renaming the over variables in this class 😄

Reply:
Done

Reply:
Noted 👍🏻